api/src/org/apache/cloudstack/api/command/user/vm/StopVMCmd.java

```diff
@@ -54,7 +54,7 @@
     private Long id;
 
     @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force stop the VM "
-        + "(vm is marked as Stopped even when command fails to be send to the backend).  The caller knows the VM is stopped.")
+        + "(vm is marked as Stopped even when command fails to be send to the backend, otherwise a force poweroff is attempted).  The caller knows the VM is stopped.")
```

Comment:
This will be a change in behaviour. Although this is cleaner not sure if it better to add a new flag for hard shutdown in order to maintain backward compatibility?